### DISCUSSION TITLE
Did you get anything from Microsoft Rewards

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#14] [General]
I remember you said a while back that you use edge and have microsoft points, and I mentioned that you could get stuff from those points at https://rewards.microsoft.com/... Did you manage to get anything you wanted there?

**Comment by TechStudent10** [comment]
tbh I totally forgot about that

**Reply by TechStudent10** [reply]
But...
![image](https://user-images.githubusercontent.com/76978184/149327350-85aac7e3-722e-4d04-96be-6342d30e5b85.png)

**Reply by KendallDoesCoding** [reply]
Send a full ss lol, you can redeem the rewards on the other screen, I think maybe its at https://rewards.microsoft.com/redeem/

**Reply by KendallDoesCoding** [reply]
Depending on which state you live you can get custom offers, like in US you can get a 20$ Amazon Card, and a Dunkin Donuts card, and if you play games like roblox even a like 20 or 100 robux, I forgot how much robux but you can get cool stuff like that with points.

**Reply by KendallDoesCoding** [reply]
![image](https://user-images.githubusercontent.com/84712013/149352468-2dc19c9f-186a-4c72-ade9-703d851eb05d.png)
![image](https://user-images.githubusercontent.com/84712013/149352545-2bb2caa0-9233-4600-87c3-d70af29746c9.png)
There are some mc stuff too

**Reply by TechStudent10** [reply]
I only care about Minecraft lol

**Reply by TechStudent10** [reply]
![image](https://user-images.githubusercontent.com/76978184/149354574-fe698be6-99ca-4525-a69a-a34be2bebfba.png)
Holy. I almost have 700 points and this is only 200.

**Reply by TechStudent10** [reply]
WAIT WHAT
![image](https://user-images.githubusercontent.com/76978184/149354819-f3ffa788-b6ca-4ef6-b771-34a239bb5a4d.png)

**Reply by TechStudent10** [reply]
6.3K WHAT THE

**Comment by KendallDoesCoding** [comment]
I suggest you dont do the sweapstakes coz like thousands of people are
trying, do something you will get things for. If you care for minecraft do
you have the paid PE?

**Comment by KendallDoesCoding** [comment]
Yes you got 6 thousand points which is a lot and can get lots of stuff.

**Comment by KendallDoesCoding** [comment]
This stuff isnt available in my country, this stuff mostly exists cause by searching for stuff on edge, doing their bing quizzes your spending time on edge and their getting money from it so they give this stuff back, but yeah for game stuff its for very less amount of points for amazon and restaurants its expensive points wise.

**Reply by TechStudent10** [reply]
Well, I guess that's true. I use Edge as my main browser and have been ever since April of 2020. When I saw that Bing was the default search engine, I told myself "You gotta switch it to Google" and never did for whatever reason.

**Reply by KendallDoesCoding** [reply]
Ok lol

**Comment by KendallDoesCoding** [comment]
So, let me know if your gonna use this otherwise, I'll have to bother you if your not to get something for me and for my friends for a couple of points cause this is not available in my country 😂. But, that's only if you dont wanna use it lol, you've earned these points fair and square.

**Reply by TechStudent10** [reply]
maybe I will idk

**Reply by KendallDoesCoding** [reply]
ok.

**Reply by KendallDoesCoding** [reply]
You used any points?

**Reply by TechStudent10** [reply]
> You used any points?

Nah they're just sitting there. I recently switched browsers from MS Edge to Brave, and the default engine is Google. So I haven't used Bing in a while.

**Comment by KendallDoesCoding** [comment]
Brave Rewards lol. Anyways, then can you take out some stuff and forward
the email it sends to me?

**Comment by KendallDoesCoding** [comment]
How many points you got again.

Also btw, go to my portfolio on brave if you want, you will see I'm a
verified creator under Brave Rewards, even though I only used brave for a
week lol

**Comment by KendallDoesCoding** [comment]
If your down, 
With 6k points, you can get 4 roblox gift cards, I want them for my friends, I don't really play the game lol, If you got 7.5k you can get 5 gift cards

URL: https://rewards.microsoft.com/redeem/000400000343

**PLEASE SELECT 100 ROBUX FOR 1500 POINTS. AFTER REDEEMEED YOU WILL GET A EMAIL, PLEASE FORWARD THE EMAIL TO kendall@kendalldoescoding.gq... Thanks a lot**

**Reply by TechStudent10** [reply]
nah i'll probably keep them lol. i probably won't use it, but maybe i will. sorry

**Comment by KendallDoesCoding** [comment]
yeah ofc, i just asked if your down/not gonna use it lol, but its your
points you can do whatever you want

**Comment by KendallDoesCoding** [comment]
> nah i'll probably keep them lol. i probably won't use it, but maybe i will. sorry

If you don't wanna wipe out the entire 6k and fine with wiping out 1.5k that's good too lol, just get one gift card. coz you can get everything under 3k max, unless you want a Amazon or Dunkin Donuts gift card, which I don't think you want. Your only interested in Minecoins the last I heard from ya, but I'm not forcing you, do this only if you want.. If you do it, you will have 4500 points left!

**Reply by TechStudent10** [reply]
well okay...

**Comment by KendallDoesCoding** [comment]
again... its your wish lol, just asking coz you got the points and its not
available in my friends/my country
